presto-accumulo/src/main/java/com/facebook/presto/accumulo/conf/AccumuloSessionProperties.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.accumulo.conf;
+
+import com.facebook.presto.spi.ConnectorSession;
+import com.facebook.presto.spi.session.PropertyMetadata;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Inject;
+
+import java.util.List;
+
+import static com.facebook.presto.spi.session.PropertyMetadata.booleanSessionProperty;
+import static com.facebook.presto.spi.session.PropertyMetadata.doubleSessionProperty;
+import static com.facebook.presto.spi.session.PropertyMetadata.integerSessionProperty;
+import static com.facebook.presto.spi.session.PropertyMetadata.stringSessionProperty;
+
+/**
+ * Class contains all session-based properties for the Accumulo connector.
+ * Use SHOW SESSION to view all available properties in the Presto CLI.
+ * <br>
+ * Can set the property using:
+ * <br>
+ * <br>
+ * SET SESSION &lt;property&gt; = &lt;value&gt;;
+ */
+public final class AccumuloSessionProperties
+{
+    private static final String INT_OPTIMIZE_LOCALITY_ENABLED = "optimize_locality_enabled";
+    private static final String INT_OPTIMIZE_SPLIT_RANGES_ENABLED = "optimize_split_ranges_enabled";
+    private static final String INT_OPTIMIZE_INDEX_ENABLED = "optimize_index_enabled";
+    private static final String INT_INDEX_ROWS_PER_SPLIT = "index_rows_per_split";
+    private static final String INT_INDEX_THRESHOLD = "index_threshold";
+    private static final String INT_INDEX_LOWEST_CARDINALITY_THRESHOLD =
+            "index_lowest_cardinality_threshold";
+    private static final String INT_INDEX_METRICS_ENABLED = "index_metrics_enabled";
+    private static final String INT_SCAN_USERNAME = "scan_username";
+
+    public static final String OPTIMIZE_LOCALITY_ENABLED =
```

Comment:
This is unused as are a bunch of others. IntelliJ has a warning for this and other inspections. Could you run them over all the code?